### DISCUSSION TITLE
min_ratio / beta 13.0beta

### DISCUSSION THREAD

**Discussion by MrSiO** [blakeblackshear/frigate#9015] [Ask A Question]
Hi.
I'm trying to get the min_ratio setting on person to get less false positive.

I've set person min_ratio to 0.5 (so that the hight is at least twice as width).

But I keep getting events with obviously a ratio larger then 1. It's confirmed also from the mqtt payload that the ratio is above 1... 

(Config and example in image)

![FB_IMG_1702955305134.jpg](https://github.com/blakeblackshear/frigate/assets/49870576/b77736e4-78c6-47d4-88f1-3a3ff30433f6)

![FB_IMG_1702955309158.jpg](https://github.com/blakeblackshear/frigate/assets/49870576/b8680fce-58af-42ba-b36f-fe476cfc985e)

Am I missing something?
Thx!

**Comment by NickM-27** [comment]
> But I keep getting events with obviously a ratio larger then 1. 

Right because min_ratio only filters out detections that are taller than that. It sounds like what you actually want is max_ratio

**Reply by MrSiO** [reply]
I might of seen it the other way around (thus the wrong way)... I think your right!

Thinking out loud:

If my min_ratio is 0.5 (hight twice it's width) then I only accept 0.51 and >... Leaning more towards a square which ain't what I want! 

Thx!
I'll fix it now!

_Suggestion: maybe a chart for average min and max ratio per listed object could be nice in the documentation?_
